content/en/cloud_cost_management/multisource_querying/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Multisource Querying
+is_beta: true
+description: Learn how to use Multisource Querying to query costs across multiple providers in Cloud Cost Management.
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Learn about Cloud Cost Management"
+- link: "/cloud_cost_management/container_cost_allocation"
+  tag: "Documentation"
+  text: "Learn about Container Cost Allocation"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Cloud Cost Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning">Multisource Querying is in private beta. To request access, contact your Customer Success Manager.</div>
+
+## Overview
+
+After you start ingesting your [AWS][1], [Azure][2], [Google Cloud][3], [SaaS][4], or [Datadog costs][5] in [Cloud Cost Management][6], you can flexibly query costs across providers. Multisource Querying enables you to query costs across multiple providers using consistent and standardized tags instead of needing to build multiple queries for each provider.
```

Comment:
```suggestion
After you start ingesting your [AWS][1], [Azure][2], [Google Cloud][3], [SaaS][4], or [Datadog costs][5] in [Cloud Cost Management][6], you can flexibly query costs across providers. Multisource Querying enables you to query costs across multiple providers using consistent and standardized tags instead of building multiple queries for each provider.
```